realm-annotations/src/main/java/io/realm/annotations/RealmField.java

```diff
@@ -31,10 +31,22 @@
 public @interface RealmField {
 
     /**
+     *
+     * Manually set the internal name used by Realm for this field. This will override any
+     * {@link RealmNamingPolicy} set on the class or the module.
+     *
+     * @see io.realm.annotations.RealmNamingPolicy for more information about what setting the name means.
+     * @see #name()
+     */
+    String value() default "";
+
+    /**
+     *
      * Manually set the internal name used by Realm for this field. This will override any
      * {@link RealmNamingPolicy} set on the class or the module.
      *
      * @see io.realm.annotations.RealmNamingPolicy for more information about what setting the name means.
+     * @see #value()
```

Comment:
In `RealmClass.java` the Javadoc is on the `name` field, we should probably follow it for consistency